typesense-java/src/integration-test/java/org/typesense/TypesenseClientIntegrationTest.java

```diff
@@ -16,14 +16,14 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.greaterThan;
-
+/*
 public class TypesenseClientIntegrationTest {
 
     private static final String TEST_COLLECTION_NAME = "intbooks";
 
     private static final Logger logger = LoggerFactory.getLogger(TypesenseClientIntegrationTest.class);
 
-    private static final int APPLICATION_PORT = 8108;
+    private static final int APPLICATION_PORT = 3001;
```

Comment:
Let's stick with `8108`.

Reply:
Seems to be the standard yep. Could probably do with an override via an environment variable or property at some stage just to be flexible.

Reply:
I haven't had time to wrap my head around the changes as a whole with the Api stuff but otherwise looks good to me.

Reply:
There's a Java Docker library that can fire up various Docker containers which we could look at too, instead of relying on a manually started Typesense server, might be a nice addition.

Reply:
There was a problem to run the server at 8108 in my machine. I will change that...No issues